### DISCUSSION TITLE
disableFocusLock description

### DISCUSSION THREAD

**Discussion by tomasp92** [elrumordelaluz/reactour#384] [Ideas]
While working with your tour I found that i needed this tool and the explanation did not help me understand its function. In lucky guess I discovered that this tool was what I was looking for. I would suggest a clearer explanation of the tool. Such as: 
"enable writing and interacting with the focused element while tour is open"

**Comment by elrumordelaluz** [comment]
Hi @tomasp92,

> The Tour uses FocusScope in order to lock the focus iteration inside the Popover when Tour is active. This prop allows to disable this behaviour.

The description is [here](https://github.com/elrumordelaluz/reactour/tree/master/packages/tour#disablefocuslock-boolean). Let me know if is still not clear and will try to improve.

Thanks,
